### PR TITLE
docs(dropdown): add demo for autoClose property

### PR DESCRIPTION
**Pull request by ghost**
Add demo for autoClose property  http://joxi.ru/KAxBo5RT4lapjm 
Related to #3781

### CODE REVIEW

File: demo/src/app/components/+dropdown/demos/autoclose/autoclose.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<div class="row">
+  <div class="col-xs-12 col-12 col-sm-6 mb-3">
+    <p>Default behavior is: not to close dropdown out of the dropdown or after pressing ESC </p>
+    <div class="btn-group" dropdown [autoClose]="false">
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/demos/autoclose/autoclose.html]: autoClose false is definitely not the default behavior. The first dropdown shouldn't use `autoClose` at all, so remove this `[autoClose]="false"`. The second dropdown must have `[autoClose]="false"` and a corresponding description

File: demo/src/app/components/+dropdown/demos/autoclose/autoclose.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<div class="row">
+  <div class="col-xs-12 col-12 col-sm-6 mb-3">
+    <p>Default behavior is: not to close dropdown out of the dropdown or after pressing ESC </p>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/demos/autoclose/autoclose.html]: Default behavior: dropdown closes after outside click or pressing ESC

File: demo/src/app/components/+dropdown/demos/autoclose/autoclose.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        <li role="menuitem"><a class="dropdown-item" href="#">Another action</a></li>
+        <li role="menuitem"><a class="dropdown-item" href="#">Something else here</a></li>
+        <li class="divider dropdown-divider"></li>
+        <li role="menuitem"><a class="dropdown-item" href="#">Separated link</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class="col-xs-12 col-12 col-sm-6 mb-3">
+    <p>Dropdown will be closed on item or document click, and after pressing ESC in this example</p>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/demos/autoclose/autoclose.html]: This dropdown won't be closed after outside click or pressing ESC

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+dropdown/dropdown-section.list.ts [hunk 4]
```diff
@@ -19,6 +19,7 @@ import {
   NgApiDocComponent,
   NgApiDocConfigComponent
 } from '../../docs/api-docs';
+import {DemoDropdownAutoCloseComponent} from "./demos/autoclose/autoclose";
```
Comment (IlyaSurmay) [inline demo/src/app/components/+dropdown/dropdown-section.list.ts]: import { DemoDropdownAutoCloseComponent } from './demos/autoclose/autoclose';

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — DISMISSED** [review]

**Pull request merged**